romancal/outlier_detection/tests/test_outlier_detection.py

```diff
@@ -293,3 +293,67 @@ def test_outlier_do_detection_find_outliers(tmp_path, base_image, clean_up_after
     assert all(o == i for i, o in zip(outliers_input_coords, outliers_output_coords))
 
     clean_up_after_test("*.asdf")
+
+
+def test_outlier_do_detection_do_not_find_outliers_in_identical_images(
```

Comment:
Just trying to understand, this looks like a reasonable but conceptually unrelated test to the main single = True bug fix above.  Is that the case?

Reply:
Yes, that's correct. In this case, we're passing copies of the same image to the method and asserting that no pixel is flagged as outlier after processing.